### PR TITLE
[AIRFLOW-4659] Fix pylint problems for api module

### PR DESCRIPTION
**Pull request by potiuk**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-4659
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
Fixes Pylint problems for api module

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
No need.
### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: airflow/api/common/experimental/trigger_dag.py [hunk 1]
```diff
@@ -16,31 +16,45 @@
...
+        dag_id: str,
+        dag_bag: DagBag,
+        dag_run: DagModel,
+        run_id: Optional[str],
+        conf: Optional[Union[dict, str]],
+        execution_date: Optional[datetime],
+        replace_microseconds: bool,
+) -> List[DagRun]:
+    """
+        Triggers DAG run.
```
Comment (mik-laj) [inline airflow/api/common/experimental/trigger_dag.py]: Unexpected indent

Comment (potiuk) [reply airflow/api/common/experimental/trigger_dag.py]: Fixed. Thanks!

**Comment by BasPH** [comment]
LGTM.

@potiuk did you know you can write Pylint disables on the same line as the code?

```python
# pylint: disable=not-callable
resp = getattr(requests, method.lower())(**params)
# pylint: enable=not-callable
```

Equivalent:
```python
resp = getattr(requests, method.lower())(**params)  # pylint: disable=not-callable
```

(in the 2nd case the disable is applied only to that single line, no need to enable again)

File: airflow/api/common/experimental/mark_tasks.py [hunk 2]
```diff
@@ -37,22 +37,23 @@ def _create_dagruns(dag, execution_dates, state, run_id_template):
...
         )
-        drs.append(dr)
-
-    return drs
+        dag_runs.append(dag_run)
 
+    return dag_runs
 
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-locals
```
Comment (BasPH) [inline airflow/api/common/experimental/mark_tasks.py]: You could also write this as a one-liner: `# pylint: disable=too-many-arguments,too-many-locals`

**Comment by potiuk** [comment]
Thanks for the hints @BasPH -  I will apply the optimised disables ;)

**Comment by potiuk** [comment]
Updated to one-liners :D

**Pull request merged**